packages/fluentui/react-northstar/test/specs/utils/positioner/positioningHelper-test.ts

```diff
@@ -69,43 +69,4 @@ describe('positioningHelper', () => {
       expectedPlacement: 'left-end',
     });
   });
-
-  describe('positioningHelper offset argument transformed correctly in RTL', () => {
```

Comment:
Please why have you removed all the tests for applyRtlToOffset?

Reply:
Oops, should restored

Reply:
Tests were updated